kolibri/plugins/style_guide/assets/src/app.js

```diff
@@ -3,6 +3,9 @@ const Vue = require('kolibri.lib.vue');
 const RootVue = require('./views');
 const router = require('kolibri.coreVue.router');
 const { navigationMenuRoutes } = require('./views/shell/navigation-menu');
+const Vuep = require('./third_party/vuep/vuep');
```

Comment:
@chaowentan I replaced this line with `require('vuep')` instead, and it worked, as long as I had installed the `codemirror` peer dependency alongside `vuep`. Could we do this instead of putting a large dist file in the code base?